presto-main/src/main/java/com/facebook/presto/operator/aggregation/AggregationFromAnnotationsParser.java

```diff
@@ -147,14 +136,25 @@ public static ParametricAggregation parseFunctionDefinition(Class<?> aggregation
         return Optional.of(getOnlyElement(stateSerializerFactories));
     }
 
-    private static AggregationHeader parseHeader(Class<?> aggregationDefinition)
+    private static AggregationHeader parseHeader(AnnotatedElement aggregationDefinition)
     {
         AggregationFunction aggregationAnnotation = aggregationDefinition.getAnnotation(AggregationFunction.class);
         requireNonNull(aggregationAnnotation, "aggregationAnnotation is null");
         return new AggregationHeader(aggregationAnnotation.value(), parseDescription(aggregationDefinition), aggregationAnnotation.decomposable());
     }
 
-    private static List<String> getNames(@Nullable Method outputFunction, AggregationFunction aggregationAnnotation)
+    private static List<AggregationHeader> parseHeaders(AnnotatedElement aggregationDefinition, AnnotatedElement toParse)
+    {
+        AggregationFunction aggregationAnnotation = aggregationDefinition.getAnnotation(AggregationFunction.class);
+
+        return getNames(toParse, aggregationAnnotation).stream().map(name ->
+            new AggregationHeader(
+                    name,
+                    parseDescription(aggregationDefinition, toParse),
+                    aggregationAnnotation.decomposable())).collect(toImmutableList());
```

Comment:
Format as
```
        return getNames(toParse, aggregationAnnotation).stream()
                .map(name -> new AggregationHeader(
                        name,
                        parseDescription(aggregationDefinition, toParse),
                        aggregationAnnotation.decomposable()))
                .collect(toImmutableList());
```

Rules:
* Nothing comes after `stream()` on the same line.
* Every operation on the stream goes in a single line.